src/scripts/main.js

```diff
@@ -1,3 +1,109 @@
 'use strict';
 
-// write code here
+const TBODY = document.querySelector('.field tbody');
+const ROWS = TBODY.rows;
+
+const APPEND_ROW_BUTTON = document.querySelector('.append-row');
+const REMOVE_ROW_BUTTON = document.querySelector('.remove-row');
+const APPEND_COLUMN_BUTTON = document.querySelector('.append-column');
+const REMOVE_COLUMN_BUTTON = document.querySelector('.remove-column');
+
+const getTableSize = () => {
+  const rows = TBODY.rows;
```

Comment:
Code duplication: You are redefining the `rows` variable here, which is already defined globally. Avoid duplicating code as it can lead to errors and makes the code harder to maintain.